libs/designsystem/src/lib/components/fab-sheet/fab-sheet.component.ts

```diff
@@ -62,7 +61,7 @@ export class FabSheetComponent implements AfterContentInit, AfterViewInit {
       const closeIcon = fabButtonElement.shadowRoot.querySelector('.close-icon ion-icon');
       if (closeIcon) {
         const closeIconSvgLoaded = closeIcon.shadowRoot.querySelector('.icon-inner svg');
-        const ionCloseIcon = (closeIcon as unknown) as IonIcon;
+        const ionCloseIcon = closeIcon as unknown as IonIcon;
```

Comment:
I did not know you could chain `as`s (so to speak 😁)

I think I like it better _with_ the parenthesises.

Is there any technical difference? How are the precedence rules for multiple `as`s?

Reply:
Nice catch, the typescript docs uses parentheses, so I am unsure of the technical differences. https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions

I'll use // prettier-ignore on this specific line.